src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function(a, b) {
+      return a + b;
+    },
+
+    subtract: function(a, b) {
+      return a - b;
+    },
+
+    multiply: function(a, b) {
+      return a * b;
+    },
+
+    divide: function(a, b) {
```

Comment:
The divide method should only take one argument and divide the current result by it. 🧮